cli/CHANGELOG.md

```diff
@@ -21,10 +25,10 @@ _Released 04/11/2023 (PENDING)_
 
 ## 12.9.0
 
-_Released 03/28/2023_
-
 **Features:**
 
+_Released 03/28/2023_
+
```

Comment:
Should these lines have moved? Bad merge?

Reply:
whoops, yeah something weird, changelog diff makes sense now